front/src/modules/ui/filter-n-sort/components/SortAndFilterBar.tsx

```diff
@@ -136,7 +138,7 @@ function SortAndFilterBar<SortField>({
   }
 
   if (
-    (!filtersWithDefinition.length && !sorts.length) ||
+    (!canToggle && !filtersWithDefinition.length && !sorts.length) ||
```

Comment:
Ok as it fixes the issue but I feel we might need to re-think this "SortAndFilterBar" to actually be a "ViewBar" that would be aware of everything: filters, sorts and views and that would force toggle itself based on filters, views and sorts. Looks too ambitious for the release so what you are doing is the way to go for now :)